index.html

```diff
@@ -0,0 +1,168 @@
+    </div>
+    <div class="card1-content">
+        <h3 class="work-title">Multy-Post Stories</h3><br><br>
+        <div class="clients">
+          <h4>CANOPY</h4>
+          <img class="img-clients" src="images/works/Counter.svg" alt="">
+          <h4 style="color: #6b778c;">Back End Dev</h4>
+          <img class="img-clients" src="images/works/Counter.svg" alt="">
+          <h4 style="color: #6b778c">2015</h4>
+        </div>
+        <p class="highlights">
+          A daily selection of privately personalized reads no accounts for
+          sign-ups required.
+        </p>   
+        <button class="tag1">HTML</button>
+        <button class="tag2">CSS</button>
+        <button class="tag3">Javascript</button>
+      
+        <button class="project-detail">See Project</button>
+    </div>
+  </section>
+
+  <!-- Second project -->
+  <section class="work-page-left">
+    <div class="card2-image">
+      <img
+      src="images/works/multi_post_story/cover.svg"
+      alt="Poster for proffesional art printing"
+      style="width: 100%;"
+    />
+    </div>
+    <div class="card2-content">
+      <h3 class="work-title">Tonic</h3>
+      <div class="clients">
+        <h4>CANOPY</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">Back End Dev</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">2015</h4>
+      </div>
+        <p class="highlights">
+          A daily selection of privately personalized reads no accounts for
+          sign-ups required.
+        </p>
+        <button class="tag1">HTML</button>
+        <button class="tag2">CSS</button>
+        <button class="tag3">Javascript</button>
+      
+        <button class="project-detail">See Project</button>
+    </div>
+  </section>
+
+    <!-- Third project -->
+  <section class="work-page-right">
+   <div class="card3-image">
+    <img src="/images/works/multi_post_story/ProfessionalArt_printing.svg" alt="Poster for Uber navigation" style="width: 100%;"/>
+   </div>
+    <div class="card3-content">
+      <h3 class="work-title">Multy-Post Stories</h3><br><br>
+      <div class="clients">
+        <h4>CANOPY</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">Back End Dev</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">2015</h4>
+      </div>
+        <p class="highlights">
+          A daily selection of privately personalized reads no accounts for
+          sign-ups required.
+        </p>   
+          <button class="tag1">HTML</button>
+          <button class="tag2">CSS</button>
+          <button class="tag3">Javascript</button>
+
+        <button class="project-detail">See Project</button>
+    </div>
+  </section>
+
+<!-- About myself section -->
+<section id="about-me" class="about-myself">
```

Comment:
Please stick to the design and fix the right and left of the beginning of the about myself section. There is no border-radius on the left, and there must be more border-radius on the right.
<img width="924" alt="Screenshot 2023-01-13 at 12 26 35 AM" src="https://user-images.githubusercontent.com/25887253/212184999-9c60c465-7d1d-4c70-bbf1-71210041b52d.png">
<img width="382" alt="Screenshot 2023-01-13 at 12 26 55 AM" src="https://user-images.githubusercontent.com/25887253/212185002-0c4f997a-c64a-4bc8-b935-7c0b12b3fb8e.png">